shared/haml/tool_explorer_table.haml

```diff
@@ -7,13 +7,14 @@
 - tools << { name: "Pixelation widget", starts: 9, ends: 12, link: "/educate/csp/widgets#pixelation", description: "Learn about graphics foramts with this interactive widget.", long: "The pixelation widget gives students the chance to learn about how images are encoded with pixel data. Then they'll encode their own images in binary.", img: "/shared/images/fill-140x140/courses/logo_pixelation.png" }
 - tools << { name: "Cryptography widgets", starts: 9, ends: 12, link: "/educate/csp/widgets#cryptography", description: "Learn about cryptography with this interactive widget.", long: "These two widgets let students play with some classic schemes for both encrypting and \"cracking\" secret messages.", img: "/shared/images/fill-140x140/courses/logo_cryptography.png" }
 - tools << { name: "Internet simulator", starts: 9, ends: 12, link: "/educate/csp/widgets#internet-simulator", description: "A group of students can learn how the internet works by communciating with each other.", img: "/shared/images/fill-140x140/courses/logo_netsim.png" }
+- tools << { name: "Game Lab (beta)", starts: 9, ends: 12, link: "/educate/gamelab", description: "Create a game or animation.", long: "Game Lab is a programming environment where you can make simple animations and games with objects and characters that interact with each other.", img: "/shared/images/fill-140x140/courses/logo_gamelab.png" }
```

Comment:
Note that this image just got changed in https://github.com/code-dot-org/code-dot-org/pull/15653.  Perhaps we want to bring it back under a different name?